src/pybind/mgr/dashboard/openapi.yaml

```
@@ -8898,6 +8898,50 @@ paths:
       - jwt: []
       tags:
       - Service
+    put:
+      description: "\n        :param service_spec: The service specification as JSON.\n\
+        \        :param service_name: The service name, e.g. 'alertmanager'.\n   \
+        \     :return: None\n        "
```

We should definitely improve the code that takes docstrings and [print those as heredoc blocks](https://stackoverflow.com/questions/8640959/how-can-i-control-what-scalar-form-pyyaml-uses-for-my-data/):
```suggestion
      description: |
      :param service_spec: The service specification as JSON.
      :param service_name: The service name, e.g. 'alertmanager'.
      :return: None
```

---

WIll create a tracker and do this imrpovement for the entire openapi.yaml file.